confd/templates/floodlight/floodlightkilda.properties.tmpl

```diff
@@ -80,3 +80,6 @@ org.openkilda.floodlight.switchmanager.SwitchManager.connect-mode=AUTO
 org.openkilda.floodlight.switchmanager.SwitchManager.broadcast-rate-limit=200
 org.openkilda.floodlight.switchmanager.SwitchManager.unicast-rate-limit=200
 org.openkilda.floodlight.switchmanager.SwitchManager.disco-packet-size=250
+org.openkilda.floodlight.switchmanager.SwitchManager.flow-meter-burst-coefficient=0.1
+org.openkilda.floodlight.switchmanager.SwitchManager.flow-meter-min-burst-size-in-kbits=1024
+org.openkilda.floodlight.switchmanager.SwitchManager.system-meter-burst-size-in-kbits=1024
```

Comment:
system rules rate are set in pktps... it will be more natural to set burls in pktps for them too.

Reply:
will fix